config/logger/logger.js

```diff
@@ -0,0 +1,30 @@
+// Package imports
+const {createLogger, format, transports} = require('winston');
+
+const logger = createLogger({
+  level: 'info',
+  format: format.combine(
+      format.timestamp({
+        format: 'YYYY-MM-DD HH:mm:ss',
```

Comment:
```suggestion
        format: 'YYYY-MM-DD HH:mm:ss',
```
Format should be referenced from constants

Reply:
Done